portfolio/src/main/webapp/learn.js

```diff
@@ -0,0 +1,54 @@
+// More API functions here:
+// https://github.com/googlecreativelab/teachablemachine-community/tree/master/libraries/image
+
+// the link to your model provided by Teachable Machine export panel
+
+const URL = "https://teachablemachine.withgoogle.com/models/7DRJsYZUY/";
+
+let model, webcam, labelContainer, maxPredictions;
+
+// Load the image model and setup the webcam
+async function init() {
+  const modelURL = URL + "model.json";
+  const metadataURL = URL + "metadata.json";
+
+  // load the model and metadata
+  // Refer to tmImage.loadFromFiles() in the API to support files from a file picker
+  // or files from your local hard drive
+  // Note: the pose library adds "tmImage" object to your window (window.tmImage)
+
+  model = await tmImage.load(modelURL, metadataURL);
+  maxPredictions = model.getTotalClasses();
+
+  // Convenience function to setup a webcam
+  const flip = true; // whether to flip the webcam
+  webcam = new tmImage.Webcam(200, 200, flip); // width, height, flip
```

Comment:
i know this was copied form the sample code, but let's make it more readable!

` webcam = new tmImage.Webcam(200, 200, flip); // width, height, flip`
 --> ` webcam = new tmImage.Webcam(/* width= */ 200, /* height= */ 200, flip);`

https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments